docs/source/postprocessing/openPMD.rst

```diff
@@ -0,0 +1,6 @@
+.. _pp-openPMD:
+
+openPMD
+=======
+
+Please see https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Post-processing-and-Visualization for now.
```

Comment:
(to stick to the current link - I just wanted to make sure that this link was your intended link)